channels/cliprdr/server/cliprdr_main.c

```diff
@@ -631,8 +631,9 @@ static UINT cliprdr_server_receive_temporary_directory(CliprdrServerContext* con
 
 	memset(cliprdr->temporaryDirectory, 0, ARRAYSIZE(cliprdr->temporaryDirectory));
 
-	if (ConvertFromUnicode(CP_UTF8, 0, wszTempDir, -1, &(cliprdr->temporaryDirectory),
-	                       ARRAYSIZE(cliprdr->temporaryDirectory), NULL, NULL) < 1)
+	char* ptr = &cliprdr->temporaryDirectory[0];
+	if (ConvertFromUnicode(CP_UTF8, 0, wszTempDir, -1, &ptr, ARRAYSIZE(cliprdr->temporaryDirectory),
```

Comment:
Oh, this won't work here. Not due to this change here, but it did not work before. This was actually something, that was mentioned afair while you were on holidays.
Basically this convert call works on `stable-2.0`, while on `master`, it will (almost) always fail. Reason is the `ARRAYSIZE(cliprdr->temporaryDirectory)`, as the actual string is likely shorter than that length here.

On `stable-2.0`, `ConvertFromUnicode()` tries to find the null terminator by itself (I think there is a manual out-of-bounds check before that) and that way does not fail.

Reply:
ok, then leave this out of here.